lib/Echidna/Solidity.hs

```diff
@@ -71,6 +74,7 @@ instance Show SolException where
     NoTests              -> "No tests found in ABI"
     OnlyTests            -> "Only tests and no public functions found in ABI"
     (ConstructorArgs s)  -> "Constructor arguments are required: " ++ s
+    NoCryticCompile      -> "crytic-compile not installed. To install it, run:\n   pip install crytic-compile"
```

Comment:
should we add that Echidna expects `crytic-compile` to be on PATH to be crystal clear?

Reply:
Historically we've said that in the releases but I agree it would be good to also add it to the README

Reply:
`pip install crytic-compile` will add crytic-compile in your path (if you use `--user`, which is better IMHO, then you should add `~/.local/bin` into your `PATH`).

Reply:
Improved error message to ask for crytic-compile in PATH.